Information Commands/Information Tools/8ball.txt

```diff
@@ -0,0 +1,10 @@
+{choose:It is certain;It is decidedly so;Without a doubt;Yes definitely;You may rely on it;As I see it, yes;Most likely;Outlook good;Yes; Signs point to yes;Reply hazy try again;Ask again later;Better not tell you now;Cannot predict now;Concentrate and ask again;Don't count on it;My reply is no;My sources say no;Outlook not so good;Very doubtful}{choose1:​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​}
+{!announce {channel} {choice1}}
+{!announce {channel} {choice1}}
+{!ann$0+ounce {channel} You have to ask a question, silly!}
```

Comment:
Is this intentional or a mistake?

Reply:
Yo soz for the late reply. That is intentional.

Reply:
It's using custom flags for the commands =). The code above it tells that what to put for the command =)